### DISCUSSION TITLE
Ability to scope Storage tree at a collection level

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5647] [Feature Requests]
> Is there a way to modify Storage (Institution) tree ranks individually in the different collections?
> We have a new storage system for the exhibitions which differs from the general storage system, and we would like to reflect this change in Specify as well.

The Storage tree is currently only scoped at the Institution level. 

**Requested By:** ZsPapp from the Natural History Museum of Denmark [on the Specify Community Forum](https://discourse.specifysoftware.org/t/modify-data-tree-ranks/115/2?u=specify)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/modify-data-tree-ranks/115/3
